src/myorganization/EditOrganization.jsx

```diff
@@ -245,7 +245,18 @@ export default function EditOrganization() {
               <div><br></br></div>
               <div className="row justify-content-between">
                 <div className="col-sm-6">
-                  <Link to={{
+                {responseMessage 
+                  ?<Link to={{
+                      pathname: `/organization-portfolio`,
+                      state: { organization }
+                      }}
+                    className="btn btn-primary disabled-link"
```

Comment:
@saideepesh000 . Just one minor change. Can you please change the colour of the button when it is in disable mode? It's good if the user can tell if the button is enabled or disabled.

Reply:
Ok sure

Reply:
that will be great 👍